hips/draw/simple.py

```diff
@@ -1,5 +1,122 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
 
+import numpy as np
+from skimage import transform as tf
+from typing import List, Generator, Any
+
+from ..tiles import HipsSurveyProperties, HipsTile, HipsTileMeta
+from ..utils import WCSGeometry, compute_healpix_pixel_indices
+
 __all__ = [
+    'draw_sky_image',
+    'make_sky_image',
+    'SimpleTilePainter'
 ]
+
+
+def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any]) -> np.ndarray:
+    """Draw sky image using the simple and quick method.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tiles : List[HipsTile]
+        A list of HipsTile
+
+    Returns
+    -------
+    np.ndarray
+        Returns a numpy array containing all HiPS tiles projected onto it
+    """
+    # TODO: Fix type annotation issue
+    all_sky = np.zeros(geometry.shape)
+    for tile in tiles:
+        draw_tile = SimpleTilePainter(geometry, tile)
```

Comment:
I find it confusing if you use the name "draw_tile" for the painter, because it's not a "tile" object.
Suggest to rename to "painter" here.